.github/workflows/beekeeper.yml

```diff
@@ -49,7 +49,6 @@ jobs:
           key: ${{ runner.os }}-build-${{ hashFiles('**/go.sum') }}
       - name: Apply patches
         run: |
-          patch pkg/postage/postagecontract/contract.go .github/patches/postagecontract.patch
```

Comment:
we have to change the beekeeper batch depth values for this to work

Reply:
here? https://github.com/ethersphere/beekeeper/blob/315be37a3d24891d3494b15d30d46aa6214ae7ef/pkg/bee/postage.go#L11